proj/src/main/webapp/chat.js

```diff
@@ -478,7 +488,7 @@ function addMoreMessagesAtTheTop() {
                     const messageUid = child.val().senderUID;
 
                     if (!blockedUsers[messageUid] && child.key !== firstChild.id) {
-                        const message = createMessageWithTemplate(child.key, child.val());
+                        const message = createMessageWithTemplate(child.key, child.val(), currentUid, 9999999999999);
```

Comment:
Hmm.  I'm not sure what Arinze thinks, but I think it would be cleaner to use a value like 0 and special-case that in the logic, rather that use a solution that will break in...hmm...November 2286.  That could also be specified as the default argument of the function, since I think the "default" case is not to mark messages unread.

Reply:
I agree with Matt. I had an experience where we wrote some code once thinking something along the lines of, this'll probably change by 2018 or 2019 or something and then that date came and it didn't change and then all our test cases randomly started failing and it took so much time to figure out why things were working one day and then just suddenly stopped.